lib/web_console/railtie.rb

```diff
@@ -55,9 +55,9 @@ class Railtie < ::Rails::Railtie
       end
     end
 
-    initializer 'web_console.whiny_requests' do
-      if config.web_console.key?(:whiny_requests)
-        Middleware.whiny_requests = config.web_console.whiny_requests
+    initializer 'web_console.verbose_logging' do
+      if config.web_console.key?(:verbose_logging)
```

Comment:
I thought it was not documented like `acceptable_content_types `, but I was wrong. Although I don't like the name, we shouldn't rename it without previous advise to users.

Will revert this change